### DISCUSSION TITLE
HandBrake

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#30] [Requests]
Can we try HandBrake next: https://handbrake.fr/

**Comment by mayrinck** [comment]
> Can we try HandBrake next: https://handbrake.fr/

Can you find the other required info?

**Reply by JuliaAgostino** [reply]
> > Can we try HandBrake next: https://handbrake.fr/
> 
> Can you find the other required info?

Yeah, sure. Here is their privacy policy: https://handbrake.fr/privacy.php
